Bottom: f7b6651f7950a2356525cce7d8d238bcb4ecb209
Top:    0cba386a52bcae3ea43d0d5b308a1ce4ec1d6af2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:31:28 +0100

Refer to git, not cvs when reading a file from newer PCB.


---

diff --git a/src/file.c b/src/file.c
index 639c971..2704ede 100644
--- a/src/file.c
+++ b/src/file.c
@@ -544,7 +544,7 @@ WritePCBDataHeader (FILE * FP)
    * ************************** README *******************
    */
 
-  fprintf (FP, "\n# To read pcb files, the pcb version (or the cvs source date) must be >= the file version\n");
+  fprintf (FP, "\n# To read pcb files, the pcb version (or the git source date) must be >= the file version\n");
   fprintf (FP, "FileVersion[%i]\n", PCB_FILE_VERSION);
 
   fputs ("\nPCB[", FP);
diff --git a/src/parse_y.y b/src/parse_y.y
index bbb7255..b76e689 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -1916,7 +1916,7 @@ check_file_version (int ver)
     Message ("ERROR:  The file you are attempting to load is in a format\n"
 	     "which is too new for this version of pcb.  To load this file\n"
 	     "you need a version of pcb which is >= %d.  If you are\n"
-	     "using a version built from cvs sources, the source date\n"
+	     "using a version built from git source, the source date\n"
 	     "must be >= %d.  This copy of pcb can only read files\n"
 	     "up to file version %d.\n", ver, ver, PCB_FILE_VERSION);
     return 1;
